contracts/KinTokenSale.sol

```diff
@@ -82,6 +80,10 @@ contract KinTokenSale is Ownable, TokenHolder {
     uint256 public lastGrantedIndex = 0;
     uint256 public constant GRANT_BATCH_SIZE = 10;
 
+    // Post-TDE multisig addresses.
```

Comment:
add `// TODO` and mention we need to set up correct addresses for this, and these are not the real addresses yet